framework/wazuh/core/exception.py

```diff
@@ -580,9 +577,7 @@ class WazuhException(Exception):
         6000: {'message': 'Limit of login attempts reached. '
                           'The current IP has been blocked due to a high number of login attempts'},
         6001: {'message': 'Maximum number of requests per minute reached',
-               'remediation': f'This limit can be changed in api.yaml file. More information here: https:/'
-                              f'/documentation.wazuh.com/{DOCU_VERSION}/user-manual/api/configuration.html#'
-                              f'configuration-file'},
+               'remediation': f'This limit can be changed in wazuh-server.yml file'},
```

Comment:
Point to the file path

```suggestion
               'remediation': f'This limit can be changed in the {WAZUH_CONFIG_PATH} file'},
```

Reply:
Fixed in 1bc09282d2b52b605beb419c08a61ba563021f0a.